WATOLINK/bci-gaming
# Alexis long range

## ClickUp ticket link

[Long Range Attack](https://app.clickup.com/t/860q9n3cu)



## Implementation description
The cowboy remians stationary and will shoot when the player becomes in range of the enemy character
It will flip the animation to whichever side the player is on so that it can detect the player on either side of it. 




## Steps to test
1. Go to the animation integration scene and press play
2. Move the player close to the cowboy character and verify that the animation plays 
3. Verify that the cowboy character flips when the player is on the other side of it


## What should reviewers focus on?
That everything works and the implementation makes sense


## Checklist
